ssl/quic/quic_wire.c

```diff
@@ -1076,3 +1077,20 @@ const char *ossl_quic_err_to_string(uint64_t error_code)
         return NULL;
     }
 }
+
+int ossl_quic_gen_rand_conn_id(OSSL_LIB_CTX *libctx, size_t len,
```

Comment:
This file seems an unexpected location for this function

Reply:
Is there a better location you have in mind?

It is somewhat miscellaneous. `quic_wire_pkt.c` is maybe better?

Reply:
Actually, since the prototype is in `quic_types.h`, I guess it really should go in a new `quic_types.c`.